docker/root/etc/s6/gitea/setup

```diff
@@ -48,6 +48,9 @@ if [ ! -f ${GITEA_CUSTOM}/conf/app.ini ]; then
     chown ${USER}:git ${GITEA_CUSTOM}/conf/app.ini
 fi
 
+# Replace app.ini settings with env variables in the form GITEA__SECTION_NAME__KEY_NAME
+environment-to-ini --config ${GITEA_CUSTOM}/conf/app.ini
```

Comment:
I don't think this merges with existing app.ini's so if an app.ini is already set then this would override it no matter what, and if no env vars are set, then it would blank things out.

Reply:
It does merge (and override) the existing app.ini. If it matches nothing it returns the config as it was. This is how I use it in my implementation - I have configured a template app.ini for any settings common to all of my deployments, and then this overrides any unique settings.

Reply:
oh, that's awesome! thanks for the clarification. In that case I only have one (maybe two) nits (haven't commented it yet), and then I'll be happy to approve :)